types/validator_set.go

```diff
@@ -718,6 +718,9 @@ func (vals *ValidatorSet) VerifyCommit(chainID string, blockID BlockID,
 // signatures.
 func (vals *ValidatorSet) VerifyCommitLight(chainID string, blockID BlockID,
 	height int64, commit *Commit) error {
+	if commit == nil {
```

Comment:
do we want to add this to `VerifyCommit` as well?

Reply:
Yeah that probably makes sense. (potentially also `VerifyCommitLightTrusting`)